x-pack/plugins/security_solution/public/management/cypress/tasks/alerts.ts

```diff
@@ -171,6 +171,8 @@ export const getEndpointDetectionAlertsQueryForAgentId = (endpointAgentId: strin
 };
 
 export const changeAlertsFilter = (text: string) => {
-  cy.getByTestSubj('queryInput').click().type(text);
-  cy.getByTestSubj('querySubmitButton').click();
+  cy.getByTestSubj('filters-global-container').within(() => {
```

Comment:
Can we use any of these two here (jntroduced in this PR)? 

export const KQL_INPUT = `${GLOBAL_KQL_WRAPPER} [data-test-subj="queryInput"]`;
export const REFRESH_BUTTON = `${GLOBAL_KQL_WRAPPER} [data-test-subj="querySubmitButton"]`;